python/hopsworks/core/variable_api.py

```diff
@@ -43,3 +66,75 @@ def get_version(self, software: str):
             if entry["software"] == software:
                 return entry["version"]
         return None
+
+    def parse_major_and_minor(self, backend_version: str) -> Tuple[Optional[str], Optional[str]]:
+        """Extract major and minor version from full version.
+
+        # Arguments
+            backend_version: The full version.
+        # Returns
+            (major, minor): The pair of major and minor parts of the version, or (None, None) if the version format is incorrect.
+        """
+
+        version_pattern = r"(\d+)\.(\d+)"
+        matches = re.match(version_pattern, backend_version)
+
+        if matches is None:
+            return (None, None)
+        return matches.group(1), matches.group(2)
+
+    def get_flyingduck_enabled(self) -> bool:
+        """Check if Flying Duck is enabled on the backend.
+
+        # Returns
+            `True`: If flying duck is availalbe, `False` otherwise.
+        # Raises
+            `RestAPIError`: If unable to obtain the flag's value.
+        """
+
+        _client = client.get_instance()
+        path_params = [
+            "variables",
+            "enable_flyingduck",
+        ]
+
+        resp = _client._send_request("GET", path_params)
+        return resp["successMessage"] == "true"
```

Comment:
just as i minor improvement i think we could do something like this (also applies for other methods in this class):
```suggestion
    def get_flyingduck_enabled(self) -> bool:
        """Check if Flying Duck is enabled on the backend.

        # Returns
            `True`: If flying duck is availalbe, `False` otherwise.
        # Raises
            `RestAPIError`: If unable to obtain the flag's value.
        """

        return self.get_variable("enable_flyingduck") == "true"
```

Reply:
but its ok if we fix it later as well, since it isn't crucial right now.

Reply:
Sounds nice, thanks!